### PR TITLE
Feature grpc status

### PR DESCRIPTION
**Pull request by ferranbt**
This PR adds a new GRPC endpoint and CLI command to return the status of the client.

### CODE REVIEW

File: docs/cli/chain.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+# Chain
+
+The ```chain``` command groups actions to interact with the blockchain in the client:
+
+- [```chain sethead```](./account_new.md): Set the current chain to a certain block.
```
Comment (victorcoder) [inline docs/cli/chain.md:6]: Is this related to this PR?

**Review by victorcoder — COMMENTED** [review]
What's the goal of knowing the current block and header? Is this info already reported using other methods?

File: docs/cli/chain.md (hunk 1 above)
Comment (ferranbt) [reply docs/cli/chain.md:6]: Nope, that is from another one I forgot to do before. Actually, the link of that command is incorrect. I will change it.

**Comment by ferranbt** [comment]
> Should we use the same keys as eth.syncing uses?

I think we have much more freedom here since it is our internal API so we can add more info about the blocks. This is just an initial PR, later on we can add more data (i.e. syncing) to this status API.

**Review by ssandeep — DISMISSED** [review]

**Review by vcastellm — APPROVED** [review]
LGTM
